src/ch/fhnw/cpib/lexer/tokens/Literal.java

```diff
@@ -7,10 +7,10 @@
     }
 
     public final Attr attr;
-    public final int numberValue;
+    public final String numberValue; // Needs to be String to support int1024
```

Comment:
I think the java iml-VM doesnt support int1024 anyways, but this change would future-proof our compiler ;)